data_inconsistencies.py

```diff
@@ -32,6 +33,35 @@ def get_inconsistencies():
     return incons
 
 
+def inconsistencies_metric(file_name):
+    f = utils.get_all_images()
+    total_img = len(f)
+    with open(file_name, 'rU') as csvfile:
```

Comment:
You are re-reading the file that is created with this script, it would be better to use the data that we have generated with `get_inconsistencies` directly